src/Server_TEST.cc

```diff
@@ -450,6 +450,42 @@ TEST_P(ServerFixture, RunNonBlocking)
   EXPECT_FALSE(*server.Running(0));
 }
 
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, RunOnceUnpaused)
+{
+    gazebo::Server server;
```

Comment:
On both tests, consider adding a system and checking that it was updated as expected.

You can use the mock system like other tests on this file.

Reply:
Sure, done in b7b09b5